03/app.js

```diff
@@ -1,3 +1,19 @@
 export default function randomNumber(min, max) {
+    if(typeof min !== 'number') {
+        throw new Error('"min" argument is not a number');
+    }
 
+    if(typeof max !== 'number') {
+        throw new Error('argument "max" is not a number');
+    }
+
+    if(min === 1 && max === 1) {
```

Comment:
Może zamiast konkretnej wartości to napisać `min === max`? Będzie lepiej działać ;)